.github/workflows/php.yml

```diff
@@ -0,0 +1,34 @@
+name: PHP Composer
+
```

Comment:
Nit: I'd remove extra newline characters to  make this more dense and readable.

Reply:
Sounds good. Came as part of the default Github workflow file for PHP.